test/torch_np/numpy_tests/core/test_multiarray.py

```diff
@@ -1856,7 +1858,7 @@ def test_searchsorted_floats(self, a):
         y = np.searchsorted(x, x[-1])
         assert_equal(y, 2)
 
-    @xpassIfTorchDynamo  # (
```

Comment:
A general note on edits like this. Related to the comment by @lezcano asking why we did not see tests which were failing start to pass.

Many of the test marked with `xpassIfTorchDynamo` were in fact only passing because a `np.dtype` call would lead to a graph break and the problematic behavior of `torch._numpy` was not observed because the normal numpy function was executed.

Now that we no longer graph break in these situations the same behavior is expected with or without `TEST_WITH_TORCHDYNAMO`.